frontend/public/components/custom-resource-definition.tsx

```diff
@@ -107,6 +113,31 @@ const Established: React.FC<{ crd: CustomResourceDefinitionKind }> = ({ crd }) =
   );
 };
 
+const CRDVersionList: React.FC<CRDVersionProps> = ({ crdVersions }) => (
+  <div className="table-responsive">
+    <table className="co-m-table-grid co-m-table-grid--bordered table">
```

Comment:
Consider making this a (non-virtualized) Patternfly table.

Reply:
@spadgett why didn't i think of that? :)